### PR TITLE
edit

### PR DESCRIPTION
**Pull request by sammychinedu2ky**
The handler for the attribute should be the `OnPostAuthorized` handler

### CODE REVIEW

**Comment by sammychinedu2ky** [comment]
@Rick-Anderson

File: aspnetcore/security/authorization/simple.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ If you decide not to use an MVC controller, the following two approaches can be
   * The [AuthorizeIndexPageHandlerFilter](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authorization/simple/samples/3.1/PageHandlerAuth/AuthorizeIndexPageHandlerFilter.cs) implements the authorization filter:
   [!code-csharp[](~/security/authorization/simple/samples/3.1/PageHandlerAuth/Pages/Index.cshtml.cs?name=snippet)]
 
-  * The [[AuthorizePageHandler]](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authorization/simple/samples/3.1/PageHandlerAuth/Pages/Index.cshtml.cs#L16) attribute is applied to the `OnGet` page handler:
+  * The [[AuthorizePageHandler]](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authorization/simple/samples/3.1/PageHandlerAuth/Pages/Index.cshtml.cs#L16) attribute is applied to the `OnPostAuthorized` page handler:
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/simple.md:107]:
```suggestion
  * The [[AuthorizePageHandler]](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authorization/simple/samples/3.1/PageHandlerAuth/Pages/Index.cshtml.cs#L28) attribute is applied to the `OnPostAuthorized` page handler:
```

**Comment by Rick-Anderson** [comment]
@sammychinedu2ky  thanks so much for this correction. I updated the correct line number in my commit.

**Comment by sammychinedu2ky** [comment]
You're welcome 😅

**Comment by Rick-Anderson** [comment]
@sammychinedu2ky oops I didn't get a chance to merge this yet. Can you restore the branch?

**Comment by Rick-Anderson** [comment]
@sammychinedu2ky  or edit in browser and copy/paste the following to line 107
```
* The [[AuthorizePageHandler]](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authorization/simple/samples/3.1/PageHandlerAuth/Pages/Index.cshtml.cs#L28) attribute is applied to the `OnPostAuthorized` page handler:
```

**Comment by sammychinedu2ky** [comment]
ooh sure I've reopened it
You can merge now. Sorry about that

File: aspnetcore/security/authorization/simple.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ If you decide not to use an MVC controller, the following two approaches can be
   * The [AuthorizeIndexPageHandlerFilter](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authorization/simple/samples/3.1/PageHandlerAuth/AuthorizeIndexPageHandlerFilter.cs) implements the authorization filter:
   [!code-csharp[](~/security/authorization/simple/samples/3.1/PageHandlerAuth/Pages/Index.cshtml.cs?name=snippet)]
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/simple.md:105]:
```suggestion
  [!code-csharp[](~/security/authorization/simple/samples/3.1/PageHandlerAuth/Pages/Index.cshtml.cs?name=snippet&highlight=21)]
```

**Comment by Rick-Anderson** [comment]
Thanks for the contribution. Be sure to check out your name as a contributor about the 3rd of next month on the [What's new page](https://docs.microsoft.com/en-us/aspnet/core/whats-new/) in the **Community contributors** section at the bottom.

**Comment by sammychinedu2ky** [comment]
Thanks @Rick-Anderson 🚀

**Pull request merged**